src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -35,13 +35,18 @@
 
   <!-- Sets the flags for the performance counters to be collected in this run. -->
   <PropertyGroup Condition="'$(PerformanceType)'=='Profile'">
-    <!-- Collect allocated bytes in execution thread, stopwatch, and any user defined Clr events (through the xUnit Performance Api attributes) -->
-    <CollectFlags>default+gcapi</CollectFlags>
+    <!-- Collect, per benchmark, execution time (stopwatch). -->
+    <CollectFlags>stopwatch</CollectFlags>
     <RunId>Perf-Profile</RunId>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(PerformanceType)'=='Diagnostic'">
-    <!-- Collect the same data collected with the Profile run plus stack and Pmc counters through Etw. -->
+    <!-- Collect, per benchmarks, the following metrics:
```

Comment:
Nit:  As above, I'd assume this should be 'Collect, per benchmark,  '

Reply:
fixed :)